contents/docs/_snippets/groups-vs-cohorts.mdx

```diff
@@ -3,4 +3,6 @@ Groups are often confused with [cohorts](/docs/data/cohorts), but they each serv
 - Groups aggregate events, and do not necessarily have to be connected to a user.
 - Cohorts represent a specific set of users – e.g., a list of users that all belong to the same company.
 
-Groups require additional code in your app to set up, while cohorts are created in PostHog and don't require additional code. This makes cohorts easier to use and quicker to get started. If your only goal is to create a **list of users** with something in common, we recommend cohorts instead of groups.
+If your only goal is to create a **list of users** with something in common, we recommend cohorts instead of groups.
```

Comment:
```suggestion
If your only goal is to analyze a **list of users** with something in common, we recommend [cohorts](/docs/data/cohorts) instead of groups.
```